includes/class-amp-travel-blocks.php

```diff
@@ -0,0 +1,199 @@
+				'amp-img'  => array_merge( $allowed_tags['img'], array(
+					'attribution' => true,
+					'class'       => true,
+					'fallback'    => true,
+					'heights'     => true,
+					'media'       => true,
+					'noloading'   => true,
+					'on'          => true,
+					'placeholder' => true,
+					'srcset'      => true,
+					'sizes'       => true,
+					'layout'      => true,
+				) ),
+				'amp-list' => array(
+					'class'            => true,
+					'credentials'      => true,
+					'placeholder'      => true,
+					'noloading'        => true,
+					'on'               => true,
+					'items'            => true,
+					'max-items'        => true,
+					'single-item'      => true,
+					'reset-on-refresh' => true,
+					'src'              => true,
+					'[src]'            => true,
+					'width'            => true,
+					'height'           => true,
+					'layout'           => true,
+					'fallback'         => true,
+				),
+			);
+
+			$allowed_tags = array_merge( $allowed_tags, $amp_tags );
+		}
+		return $allowed_tags;
+	}
+
+	/**
+	 * Register Travel Discover block type.
+	 */
+	public function register_block_discover() {
+		register_block_type( 'amp-travel/discover', array(
+			'attributes'      => array(
+				'heading'    => array(
+					'type'    => 'string',
+					'default' => __( 'Discover Adventures', 'travel' ),
+				),
+				'subheading' => array(
+					'type'    => 'string',
+					'default' => __( 'Get inspired and find your next big trip', 'travel' ),
+				),
+			),
+			'render_callback' => array( $this, 'render_block_discover' ),
+		) );
+	}
+
+	/**
+	 * Front-side render for Travel Discover block.
+	 *
+	 * @param array $attributes Block attributes.
+	 * @return string Output.
+	 */
+	public function render_block_discover( $attributes ) {
+		global $post;
+
+		$args = array(
+			'numberposts' => 1,
+			'post_status' => 'publish',
+			'post_type'   => 'post',
+		);
+
+		if ( $post ) {
+			$args['exclude'] = array( $post->ID );
+		}
+
+		$posts = wp_get_recent_posts( $args );
+
+		$heading    = $attributes['heading'];
+		$subheading = $attributes['subheading'];
+
+		// If there's no post, return.
+		if ( empty( $posts ) ) {
+			return '';
+		} else {
+			$discover_post = $posts[0];
+			$title         = get_the_title( $discover_post['ID'] );
+			$link          = get_permalink( $discover_post['ID'] );
+
+			// Use the wp_trim_words directly since otherwise the_content will run endlessly due to wp_trim_excerpt() using the global post.
+			if ( empty( $discover_post['post_excerpt'] ) ) {
+				$excerpt_length = apply_filters( 'excerpt_length', 15 );
+				$excerpt_more   = apply_filters( 'excerpt_more', ' ...' );
+				$excerpt        = wp_trim_words( $discover_post['post_content'], $excerpt_length, $excerpt_more );
+			} else {
+				$excerpt = get_the_excerpt( $discover_post['ID'] );
+			}
+		}
+
+		$output = "<section class='travel-discover py4 mb3 relative xs-hide sm-hide'>
+					<div class='max-width-3 mx-auto px1 md-px2'>
```

Comment:
Could you indent this `<section class='travel-discover` line and everything below it one tab to the left (except for the `return $output;` statement).